CMakeLists.txt

```diff
@@ -38,12 +33,23 @@ else()
 endif()
 
 # add SDK
-include_directories(${SDK_INSTALL_DIR}/include)
-link_directories(${SDK_INSTALL_DIR}/lib)
+include(ExternalProject)
```

Comment:
No need to `include(ExternalProject)` twice from the same file, I would say.

Reply:
Grmbl,  you're right.